setup.cfg

```diff
@@ -27,20 +27,19 @@ zip_safe = False
 packages = find:
 include_package_data = True
 install_requires =
-    dvc
+    dvc@git+https://github.com/iterative/dvc
```

Comment:
This will break package building. We install upstream dvc in github actions here already.